composer.json

```diff
@@ -32,18 +32,18 @@
         "doctrine/instantiator": "^1.3 || ^2",
         "doctrine/lexer": "^2.1 || ^3",
         "doctrine/persistence": "^3.1.1",
-        "psr/cache": "^1 || ^2 || ^3",
+        "psr/cache": "^3",
```

Comment:
We require the fluent interface: https://github.com/doctrine/orm/blob/62ed63bbbe3f59dcacb7dadb574895f5cb27a257/lib/Doctrine/ORM/Query.php#L276

Reply:
The fluent interface is part of the standard since v1.

Reply:
:thinking: I guess the issue must be with a wrong implementation then.

Reply:
> MockObject_CacheItemInterface_9583ebe3

Ah yes, that makes sense.

Reply:
Fixed properly.